### PR TITLE
Added new pubsub service to performance pipeline.

### PR DESCRIPTION
**Pull request by apolloakora**
# Motivation and Context
Adding the new pubsub service to the performance test pipeloine.

# What has changed
The naming of the performance tests. The more significant changes are in the kubernetes repo which has the queue definitions.

# How to test?
Run the performance test pipeline in gcloud and view the grafana dashboard for anomalies and compare the results to what is tentatively expected.

### CODE REVIEW

File: pipelines/performance-tests-pipeline.yml [hunk 1]
```diff
@@ -410,7 +410,7 @@ jobs:
     fieldwork-adapter,
     notify-processor,
     uac-qid-service,
-    pubsubsvc,
+    pubsub-adaptersvc,
```
Comment (AdamHawtin) [inline pipelines/performance-tests-pipeline.yml]: For consistency I'd rather this was just `pubsub-adapter`

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Changes make sense, just a small request for naming consistency 👍

**Comment by apolloakora** [comment]
[Replace python pubsub service with pubsub-adapter in performance pipeline (5)](https://trello.com/c/0iQgvsBW/921-replace-python-pubsub-service-with-pubsub-adapter-in-performance-pipeline-5)

**Review by AdamHawtin — APPROVED** [review]

**Review by NickGrantX — CHANGES_REQUESTED** [review]
please remove the `svc` suffix

File: pipelines/performance-tests-pipeline.yml [hunk 2]
```diff
@@ -730,10 +730,10 @@ jobs:
       WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
     input_mapping: {kubernetes-repo: census-rm-kubernetes-release}
 
-- name: "PubSub Service"
+- name: "PubSub Adapter Service"
```
Comment (NickGrantX) [inline pipelines/performance-tests-pipeline.yml]: just "PubSub Adapter"

File: pipelines/performance-tests-pipeline.yml [hunk 3]
```diff
@@ -962,7 +962,7 @@ jobs:
       "Case API",
       "Case Processor",
       "UAC QID Service",
-      "PubSub Service",
+      "PubSub Adapter Service",
```
Comment (NickGrantX) [inline pipelines/performance-tests-pipeline.yml]: Just "PubSub Adapter"

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**